framework/src/org/checkerframework/framework/util/expressionannotations/ExpressionAnnotationTreeAnnotator.java

```diff
@@ -0,0 +1,62 @@
+package org.checkerframework.framework.util.expressionannotations;
+
+import com.sun.source.tree.IdentifierTree;
+import com.sun.source.tree.MemberSelectTree;
+import com.sun.source.tree.NewArrayTree;
+import com.sun.source.tree.NewClassTree;
+import com.sun.source.tree.TypeCastTree;
+import com.sun.source.tree.VariableTree;
+import javax.lang.model.element.Element;
+import org.checkerframework.framework.type.AnnotatedTypeFactory;
+import org.checkerframework.framework.type.AnnotatedTypeMirror;
+import org.checkerframework.framework.type.treeannotator.TreeAnnotator;
+import org.checkerframework.javacutil.TreeUtils;
+
+/** Standardizes Java expressions in annotations and also view points adapts field accesses. */
```

Comment:
Why only field accesses and not also methods and constructors? Who takes care of those?